src/ert/scheduler/lsf_driver.py

```diff
@@ -146,17 +156,21 @@ async def kill(self, iens: int) -> None:
     async def poll(self) -> None:
         while True:
             if not self._jobs.keys():
-                await asyncio.sleep(_POLL_PERIOD)
+                await asyncio.sleep(self._poll_period)
                 continue
-            proc = await asyncio.create_subprocess_exec(
+            process = await asyncio.create_subprocess_exec(
                 self._bjobs_cmd,
                 *self._jobs.keys(),
                 stdout=asyncio.subprocess.PIPE,
                 stderr=asyncio.subprocess.PIPE,
             )
 
-            stdout, _ = await proc.communicate()
-            stat = _Stat(**parse_bjobs(stdout))
+            stdout, stderr = await process.communicate()
+            if process.returncode:
+                logger.error(
+                    f"bjobs gave returncode {process.returncode} and error {stderr.decode()}"
+                )
```

Comment:
`continue` ?

Reply:
yeah, that makes sense!

Reply:
No it does not make sens, as there is a test that will fail if we add continue here. I don't trust the return code from bjobs, see the test where it returns 255.

Reply:
downgrading loglevel to warning.